README.md

```diff
@@ -1,3 +1,23 @@
+##
+Updates to the Readme.md by Dean Smythe
+See diagrams in Excel document and pdf.
+Chitter_Domain_Model.xlsx
+Chitter_Domain_Model.pdf
+
```

Comment:
It's really great that you're using domain modelling as part of your planning process.  This will become particularly useful when you're working in teams (maybe you already discovered that this week!)